curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5efc4f528d6a74d05e68af74.md

```diff
@@ -70,7 +70,13 @@ The text `Loving` should be nested within the new `label` element. You have eith
 
 ```js
 const labelElem = document.querySelector('label[for="loving"]');
-assert(labelElem.textContent.replace(/\s/g, '').match(/Loving/i));
+const paragraphElem = document.querySelectorAll("fieldset");
```

Comment:
The code from the issue no longer passes, but I get this hint - which that code has. I would revert this test to what it was and change the test on line 28 and/or 48, both of which should have failed here based on their text.

Reply:
Ah, I think I was separating the <input id="loving" type="checkbox"><label for="loving">Loving</label> onto two lines to get the pass.
Example:
```<input id="loving" type="checkbox">
<label for="loving">Loving</label>```
Seems a bit odd because the spaces should be non existent / removed with the regex.
I'll have another look at it tonight and see if I can tighten a conditional check on the lines you referenced.
Thanks

Reply:
I don't know if that passed for me or not. To try and clarify, if you use the code from the issue:
```
<input id="loving" type="checkbox">Loving
<label for="loving" type="checkbox">Loving</label>
```
It doesn't pass and you get the hint from this test you were changing:
```
The text `Loving` should be nested within the new `label` element
```
That code does have the text `Loving` nested in the `label`. So people won't know why it is failing. Based on the text of the other two tests I mentioned:
```
The new `label` element should be located directly to the right of your checkbox
```
and
```
The text ` Loving` should no longer be located directly to the right of your checkbox
```

Both of those should have failed with that code. So I think we should maybe change one or both of those tests so they will fail.